plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineSdkClasspathContainer.java

```diff
@@ -18,7 +17,7 @@
   private static final String SDK_JAR = "platform/google_appengine/google/appengine/tools/java/lib/appengine-tools-api.jar";
   public static final String CONTAINER_ID = "AppEngineSDK";
 
-  private PathResolver pathResolver = PathResolver.INSTANCE;
+  private File cloudSdkLocation = new CloudSdkProvider().getCloudSdkLocation();
```

Comment:
So it looks fine to me.  I wonder if we should instead hold onto the `CloudSdk` instance and ask that instance for the SDK jar(s).